Dockerfile

```diff
@@ -1,8 +1,34 @@
 FROM nfcore/base
 MAINTAINER Phil Ewels <phil.ewels@scilifelab.se>
 LABEL authors="phil.ewels@scilifelab.se" \
-    description="Docker image containing all requirements for the nfcore/ChIPseq pipeline"
+      description="Docker image containing all requirements for the nfcore/ChIPseq pipeline"
 
 COPY environment.yml /
-RUN conda env create -f /environment.yml && conda clean -a
+RUN conda update -n base conda
+RUN conda update --all
+RUN conda env update -n root --file environment.yml  && conda clean -a
 ENV PATH /opt/conda/envs/nfcore-chipseq-1.4dev/bin:$PATH
```

Comment:
I'm not very familiar with the conda environment. But If I understand it  right we might be able to remove that line. It is better to check that before. I have a question about nexflow. Can I specify which process to reexecute (even after completed)? I removed the folder of the process in the work directory and it just restarted all processes.

Reply:
> Can I specify which process to reexecute (even after completed)? I removed the folder of the process in the work directory and it just restarted all processes.

No, it's automatic - if it thinks that downstream processes rely on a work folder that you deleted, then it will rerun them all. It does this because it can't know that the downstream results would be the same otherwise.

If I'm misunderstanding your question, then come ask on gitter where we can chat: https://gitter.im/nf-core/Lobby